<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vButacaCarga</class>
 <widget class="QWidget" name="vButacaCarga">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>113</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cinemar - Carga de Butacas</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>284</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblNroSala">
       <property name="text">
        <string>NÃºmero Sala</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtNroSala"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnCargar">
       <property name="text">
        <string>Cargar Butaca</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblFila">
       <property name="text">
        <string>Fila</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtFila"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblAsiento">
       <property name="text">
        <string>Asiento</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtAsiento"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
